Content.IntegrationTests/Tests/Destructible/DestructibleDamageTypeTest.cs

```diff
@@ -57,23 +58,24 @@ public async Task Test()
             await server.WaitAssertion(() =>
             {
 
-                var bluntDamage = sDamageableComponent.GetDamageType("Blunt");
-                var SlashDamage = sDamageableComponent.GetDamageType("Slash");
+                
+                var bluntDamageType = IoCManager.Resolve<IPrototypeManager>().Index<DamageTypePrototype>("Blunt");
+                var slashDamageType = IoCManager.Resolve<IPrototypeManager>().Index<DamageTypePrototype>("Slash");
```

Comment:
would be better if these used extra prototypes for test damage types rather than assuming these will always exist; shouldnt be hard to change

Reply:
done